api/v1alpha1/trafficmanagerbackend_types.go

```diff
@@ -75,18 +82,33 @@ type TrafficManagerEndpointStatus struct {
 	// +required
 	Name string `json:"name"`
 
+	// ResourceID is the fully qualified Azure resource Id for the resource.
+	// Ex - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/trafficManagerProfiles/{profileName}/azureEndpoints/{name}
+	ResourceID string `json:"resourceID,omitempty"`
+
 	// The weight of this endpoint when using the 'Weighted' traffic routing method.
-	// Possible values are from 1 to 1000.
+	// Possible values are from 0 to 1000.
 	// +optional
 	Weight *int64 `json:"weight,omitempty"`
 
 	// The fully-qualified DNS name or IP address of the endpoint.
 	// +optional
 	Target *string `json:"target,omitempty"`
 
-	// Cluster is where the endpoint is exported from.
+	// From is where the endpoint is exported from.
```

Comment:
why change this field name? Is there some API pattern we follow ?

Reply:
i'm trying to solve duplicate field "cluster" before v1beta1, meanwhile we're introducing two layers weight and i created a structure to include the "weight" per cluster for better visibility.

<img width="755" alt="image" src="https://github.com/user-attachments/assets/8a6f5071-4d97-4596-b7a8-1f9ae3a131c3" />